set.mm

```diff
@@ -85685,6 +85710,101 @@ the first case of his notation (simple exponentiation) and subscript it
       IJK $.
   $}
 
+  ${
+    $d B b f g $.  $d S b f g $.  $d V b g $.
+    $( The class of all functions from a (proper) singleton into ` B ` is the
+       union of all the singletons of (proper) ordered pairs over the elements
+       of ` B ` as second component.  (Contributed by AV, 13-Sep-2024.) $)
+    fsetsniunop $p |- ( S e. V -> { f | f : { S } --> B }
+                                  = U_ b e. B { { <. S , b >. } } ) $=
+      ( vg wcel csn cv wf cab cop ciun wrex wceq cfv wa fsn2g simpl simpr opeq2
+      wb sneqd eqeq2d adantl rspcedvd syl6bi fsnd adantr feq1d mpbird rexlimdva
+      ex impbid velsn bicomi rexbii bitrdi vex feq1 elab eliun 3bitr4g eqrdv )
+      BDGZFBHZACIZJZCKZEABEIZLZHZHZMZVEVFAFIZJZVOVMGZEANZVOVIGVOVNGVEVPVOVLOZEA
+      NZVRVEVPVTVEVPBVOPZAGZVOBWALZHZOZQZVTBAVODRWFVSWEEWAAWBWESVJWAOZVSWEUBWFW
+      GVLWDVOWGVKWCVJWABUAUCUDUEWBWETUFUGVEVSVPEAVEVJAGZQZVSVPWIVSQZVPVFAVLJZWI
+      WKVSWIBVJDAVEWHSVEWHTUHUIWJVFAVOVLWIVSTUJUKUMULUNVSVQEAVQVSFVLUOUPUQURVHV
+      PCVOFUSVFAVGVOUTVAEVOAVMVBVCVD $.
+
+    $d B b y $.  $d S y $.
+    $( The class of all functions from a (proper) singleton into ` B ` is the
+       set of all the singletons of (proper) ordered pairs over the elements of
+       ` B ` as second component.  (Contributed by AV, 13-Sep-2024.) $)
+    fsetabsnop $p |- ( S e. V -> { f | f : { S } --> B }
```

Comment:
I think ~fsetsniunop and ~fsetabsnop are too specific to be in Main. (If they find applications later, they can be moved from your mathbox to Main.)

Reply:
Done (moved into my mathbox)